serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc

```diff
@@ -1 +1,257 @@
-= Create your first workflow service
\ No newline at end of file
+= Create your first workflow service
+
+Learn how to create a Hello World Serverless Workflow application. This guide covers:
+
+* Bootstrapping an application
+* Creating a workflow
+* Running the application
+* Testing the application
+
+== Prerequisites
```

Comment:
Shouldn't there be prerequisites such as:

time needed to go through the guide? 10 minutes?
JDK 11?
Apache Maven minimal version?
Optionally Quarkus CLI

Reply:
Linking to the guide was a suggestion by @ricardozanini [here](https://github.com/kiegroup/kogito-docs/pull/2#discussion_r882846710).

Reply:
@ricardozanini  I think this guide should work as a starting point without the need to go to another guide. JDK 11, Maven should be the minimum in my opinion. We could say that if they are interested in more tooling they can follow the tooling guide as you suggest. WDYT?